README.rst

```diff
@@ -90,3 +90,74 @@ Architecture
      |
      |
      v
+
+
+FAQ
+------------
+
+Q: If the worker and backend complain about ::
+
+    self.db[key] = value
+    TypeError: String or Integer object expected for key, unicode found
+
+A: Modify the config for `fedmsg.d/fmn.py` to use redis ::
+
+  # Some configuration for the rule processors
```

Comment:
yea i see that as well, i was using two different editors. I'll get that sorted out soon